### ISSUE TITLE
Switching File menu items then back remembers which submenu item was last highlighted

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#6618]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.929
    OS Version      : MacOS 10.15.4 pointing at Red Hat 7.8 via Safari
    R Version       : 3.6.0

### Steps to reproduce the problem

1. On MacOS, turn VoiceOver on
2. Go to the File menu bar, e.g. with <kbd>Ctrl-Opt-F<kbd>
3. Right Arrow down to Terminate R
4. Left arrow to the Plots menu
5. Right arrow back to the Session menu

### Describe the problem in detail

VoiceOver announces that it's on the Terminate R menu item, not on the Session drop down menu.

### Describe the behavior you expected

If one highlights a menu item, switches to another, then back to the original one, the name of the menu should be stated—e.g., Session—and not the previous focused-on submenu item.

Note: This works fine with VoiceOver turned off, as well as Windows 10 with Chrome and NVDA turned on.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jooyoungseo** [comment]
@ronblum, have you tried navigating through the menus with VoiceOver cursor?

Arrow keys use system cursor; CTRL+OPT+[arrows] utilize a VoiceOver cursor.

VoiceOver cursor has its own focus so I was wondering if this has to do with that focus issue.

@gtritchie, I have concerns about having CTRL+OPT key combination for accessing menu bar on Mac OS. Because CTRL+OPT is used as a VoiceOver key globally as in Capslock is taken for NVDA key. This might be another issue we need to create though... For example, CTRL+OPT+F is originally taken by VoiceOver to bring up a "find" dialog on web pages. CTRL+OPT+U pops up an element choose list box, etc. etc.

**Comment by gtritchie** [comment]
@jooyoungseo Hi, yes I'm aware of the clash with default VoiceOver keys. User either has to configure VoiceOver to use CapsLock, or use the VO+Tab prefix before using the menu keys.

**Comment by jooyoungseo** [comment]
@gtritchie, yes, I understand CTRL+OPT key combination is kind of convension on Mac OS (and Google Docs for Mac seem utilize them...), and it would be too late to change the shortcuts given that it has been already widely used. Maybe, we could later document the recommendation of switching the VO key to Capslock for VoiceOver users.

**Comment by gtritchie** [comment]
@jooyoungseo Yes, that is the plan. I have a placeholder article on using screen reader with RStudio that I still need to author and will include this information. I will give you a draft soon to get your feedback. Thanks!
